src/Tribe/Service_Provider.php

```diff
@@ -33,6 +33,16 @@ public function register() {
 		$this->container->singleton( 'tickets.hubspot.api', API\Connection::class );
 		$this->container->singleton( 'tickets.hubspot.oauth', API\Oauth::class, array( 'hook' ) );
 
+		$this->container->singleton( 'tickets.hubspot.properties.last_registered_event', new Properties\Last_Registered_Event() );
```

Comment:
This and all classes below should just pass `::class` for each class

Reply:
Updated these.